raiden/network/transport/matrix/rtc/web_rtc.py

```diff
@@ -459,7 +460,10 @@ def _handle_message(self, message_data: str, partner_address: Address) -> None:
     def _handle_ice_connection_closed(self, conn: _RTCConnection) -> None:
         # the connecting may have already been removed by close_connection
         self._address_to_connection.pop(conn.partner_address, None)
-        self.health_check(conn.partner_address)
+
+        # only do a health check if we were the initiator
+        if conn.call_id.split("|", 1)[0] == to_checksum_address(self.node_address):
+            self.health_check(conn.partner_address)
```

Comment:
as discussed, I'd prefer a flag check instead of a string comparison of the call_id

Reply:
especially when we think about backwards compatibility with the LC, it would be more strict to not rely on the string. Also No where in the code base the checksummed address is used for any logic. The internal representation is always binary

Reply:
Changed to avoid interpreting the call ID.